docs/guide/plugins.md

```diff
@@ -0,0 +1,42 @@
+# Plugins
+
+Plugins are now supported in the application. The plugin packages are not included when npm is installing packages
+
+## Installing Plugin
+
+### NPM Method
+
+you can install a plugin using `npm i` (install) command. It is advised to run this command after building.
+
+example installing `@user/plugin-name`
+
+```bash
+npm i @user/plugin-name
+```
```

Comment:
When using plugins you can place them in a custom folder and use as plugin name the absolute path to it. Otherwise if them are published to npm you can more easily run `npm install <pluginName>` and use just `<pluginName>` in settings to use them instead of specify the absolute path.